tests/test_add_new_jobs.py

```diff
@@ -0,0 +1,26 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.treeherder import TreeherderPage
+
+
+@pytest.mark.nondestructive
+def test_add_new_jobs(base_url, selenium):
+    page = TreeherderPage(base_url, selenium).open()
+    assert len(page.result_sets) > 1
```

Comment:
Nit: Both here and on line 21, I think we can assert that there's at least one result set, as we only use the first one. You could make this `> 0` or `>= 1`.